src/utils/flatMap.js

```diff
@@ -0,0 +1,3 @@
+export default (array, mapFunction) => {
+  return Array.prototype.concat.apply([], array.map(mapFunction));
```

Comment:
Why not `return [].concat(array.map(mapFunction));`?

Reply:
Nevermind, that was a dumb question. We could do `return [].concat(...array.map(mapFunction));`, which would be a little shorter in es6 world, but babel would transpile it to something more complicated and expensive. I like yours.